moncli/entities/item.py

```diff
@@ -1,9 +1,10 @@
+from moncli.entities.column_value.base import ColumnValue
```

Comment:
When working inside of the moncli package directly, 

Please use the relative imports instead of the direct ones.